tests/python/utils.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# Owner(s): ["module: nvfuser"]
+
+import torch
+from torch.testing._internal.common_utils import TEST_WITH_ROCM
+from torch.testing._internal.jit_utils import RUN_CUDA
+
+# flake8: noqa
+from nvfuser import FusionDefinition, DataType
+
+RUN_NVFUSER = RUN_CUDA and not TEST_WITH_ROCM
+
+
+def is_pre_volta():
+    if not RUN_NVFUSER:
+        return False
```

Comment:
Should this be inside these functions? We have a separate `unittest.skip` decorator for this variable

Reply:
I'm not sure if that is true for the pytest framework.